cmd/benchcore/main.go

```diff
@@ -765,6 +765,7 @@ $HOME/corectl create-token -net benchcorenet > $HOME/network-token.txt
 
 const coredsh = `#!/bin/bash
 set -eo pipefail
+sudo bash -c "ulimit -n 65535 && exec su ubuntu"
```

Comment:
This line doesn't do anything useful. It starts a bash process, which changes its own ulimit, then execs a new process as a different user, which does nothing and then exits.